### DISCUSSION TITLE
How to set fillets, shadows, borders, and padding for widgets in giu?

### DISCUSSION THREAD

**Discussion by Seikaijyu** [AllenDang/giu#531] [Q&A] [answered]
I want to know how to set these properties to make the style more beautiful

**Comment by Seikaijyu** [comment]
I always have many problems😭

**Comment by gucio321** [comment, marked as answer]
Hmm check out a set of style flags in Style.go - that's all giu ( and as far as i know imgui) offers

**Reply by Seikaijyu** [reply]
![image](https://user-images.githubusercontent.com/68497295/179419117-a6e55808-51b0-4322-9de8-65b9d930c9ba.png)
Are all the styles in giu.Style()? I saw these flags in giu. Can they be used and how should they be used?

**Reply by gucio321** [reply]
They're are draw flags used for canvas. For styles start typing "StyleVar"

**Reply by Seikaijyu** [reply]
There is a small problem. Is giu implementing the rounded corner of the button

**Reply by Seikaijyu** [reply]
How should I cancel rounding
![image](https://user-images.githubusercontent.com/68497295/179434498-c87b14e0-49a1-4c61-933d-118746251f0d.png)

**Reply by gucio321** [reply]
Just set it to zero. Giu has its own default style that has some rounding by default.
Btw you don't need to use giu styled style setter instead just push style with giu.PushStyle and pop it at the end of loop func

**Reply by Seikaijyu** [reply]
Set what to 0? Sorry, I didn't understand

**Reply by gucio321** [reply]
Hmm iirc it's called StyleVarFrameRounding or something like that... Just giu.Style().SetStyle(giu.StyleVarFrameRounding, 0).To(...)

**Reply by Seikaijyu** [reply]
It's solved successfully. Thank you again for your patient answer
